### ISSUE TITLE
Confused about label files. 

### ISSUE THREAD

**Issue by qaazii** [lzccccc/3d-bounding-box-estimation-for-autonomous-driving#1]
Hi Author, I am confuse about the labels_02, calib_02 files in the folder 
kitti_dateset/
├── 2011_09_26
│   └── 2011_09_26_drive_0084_sync
│           ├── box_3d       <- predicted data
│           ├── **calib_02**
│           ├── calib_cam_to_cam.txt
│           ├── calib_velo_to_cam.txt
│           ├── image_02
│           ├── _**label_02**_
│           └── tracklet_labels.xml

i dont find this the kitti raw data. can you please explain where do you get this data from?

**Comment by lzccccc** [comment]
Hi,
The kitti raw data provides tracklet sequence in xml format. You need to run parse_raw_to_KITTI_form.py under /data_processing/raw_data_processing/ to change the tracklet_labels.xml to KITTI object detection txt format. Hope this can help. :)

**Comment by linchaozhang** [comment]
kitti_dateset/
├── 2011_09_26
│   └── 2011_09_26_drive_0084_sync
│           ├── box_3d       <- predicted data
│           ├── calib_02
│           ├── calib_cam_to_cam.txt
│           ├── calib_velo_to_cam.txt
│           ├── image_02
│           ├── label_02
│           └── tracklet_labels.xml
│
└── training
    ├── box_3d    <- predicted data
    ├── calib
    ├── image_2
    └── label_2
Hi , @lzccccc 
which one should i download the dataset?
http://www.cvlibs.net/datasets/kitti/eval_object.php?obj_benchmark=3d
Download left color images of object data set (12 GB)
Download right color images, if you want to use stereo information (12 GB)
Download the 3 temporally preceding frames (left color) (36 GB)
Download the 3 temporally preceding frames (right color) (36 GB)
Download Velodyne point clouds, if you want to use laser information (29 GB)
Download camera calibration matrices of object data set (16 MB)
Download training labels of object data set (5 MB)
Download object development kit (1 MB) (including 3D object detection and bird's eye view evaluation code)
Download pre-trained LSVM baseline models (5 MB) used in Joint 3D Estimation of Objects and Scene Layout (NIPS 2011). These models are referred to as LSVM-MDPM-sv (supervised version) and LSVM-MDPM-us (unsupervised version) in the tables below.
Download reference detections (L-SVM) for training and test set (800 MB)

**Comment by qaazii** [comment]
Download left color images of object data set (12 GB) for images
Download training labels of object data set (5 MB) labels
Download camera calibration matrices of object data set (16 MB) for caliberation matrix

**Comment by ghost** [comment]
Where does the 2011_09_06 directory coming from? Is it needed?
